app/src/main/java/com/larryhsiao/nyx/android/jot/JotContentFragment.java

```diff
@@ -0,0 +1,212 @@
+package com.larryhsiao.nyx.android.jot;
+
+import android.app.AlertDialog;
+import android.content.Intent;
+import android.location.Location;
+import android.net.Uri;
+import android.os.Bundle;
+import android.view.*;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.RecyclerView;
+import com.google.android.material.chip.Chip;
+import com.google.android.material.chip.ChipGroup;
+import com.larryhsiao.nyx.R;
+import com.larryhsiao.nyx.android.base.JotFragment;
+import com.larryhsiao.nyx.attachments.AttachmentsByJotId;
+import com.larryhsiao.nyx.attachments.NewAttachments;
+import com.larryhsiao.nyx.attachments.QueriedAttachments;
+import com.larryhsiao.nyx.attachments.RemovalAttachmentByJotId;
+import com.larryhsiao.nyx.jots.*;
+import com.larryhsiao.nyx.tags.*;
+import com.schibstedspain.leku.LocationPickerActivity;
+import com.silverhetch.aura.location.LocationAddress;
+import com.silverhetch.clotho.source.ConstSource;
+
+import java.util.stream.Collectors;
+
+import static android.app.Activity.RESULT_OK;
+import static android.content.Intent.ACTION_OPEN_DOCUMENT;
+import static android.content.Intent.FLAG_GRANT_READ_URI_PERMISSION;
+import static com.schibstedspain.leku.LocationPickerActivityKt.*;
+
+/**
+ * Fragment that shows the Jot content.
+ */
+public class JotContentFragment extends JotFragment {
+    private static final int REQUEST_CODE_LOCATION_PICKER = 1000;
+    private static final int REQUEST_CODE_PICK_FILE = 1001;
+    private static final String ARG_JOT_ID = "ARG_JOT_ID";
+    private ChipGroup chipGroup;
+    private EditText contentEditText;
+    private TextView locationText;
+    private ImageView attachmentIcon;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'attachmentIcon' could be replaced by a local variable.](https://app.codacy.com/manual/LarryHsiao/Nyx/pullRequest?prid=5029215)